resources/tentative/complex-static-html/src/big-dom-generator/html-markup.js

```diff
@@ -29,8 +29,8 @@ export const ANGULAR_TODO_MVC_HTML_MARKUP = `
 export const TODO_MVC_HTML_MARKUP = `
 <div class="main-ui">
     <div class="show-more"></div>
-    <div class="top-bar"/></div>
-    <div class="ribbon"/></div>
+    <div class="ribbon"></div>
```

Comment:
you can do `<div class="ribbon"/>` and remove the closing `</div>` tag.

Reply:
JSDOM doesn't handle that well. the resulting output is
```html
<div class="main-ui">
    <div class="show-more">
    <div class="ribbon">
    <div class="top-bar">
    <div class="tree-area">
    <div class="todo-area">
        <div class="todoholder">
            <section class="todoapp">
                <header class="header" data-testid="header"></header>
                <main class="main" data-testid="main">
                    <div class="toggle-all-container">
                        <input class="toggle-all" type="checkbox" data-testid="toggle-all">
                        <label class="toggle-all-label" for="toggle-all">Toggle All Input</label>
                    </div>
                    <ul class="todo-list" data-testid="todo-list"></ul>
                </main>
            </section>
        </div>
    </div>
</div>
</div></div></div></div>
```

Reply:
Good to know, thanks!